.meta/sinks/gcp_pubsub.toml

```diff
@@ -55,3 +55,5 @@ common = false
 required = true
 examples = ["this-is-a-topic"]
 description = "The topic within the project to which to publish logs."
+
+<%= render("_partials/_encoding.toml", namespace: "sinks.gcp_pubsub.options", encodings: ["default"], default: "default") %>
```

Comment:
1. `default` should never be an option.
2. Please break this out into multiple lines for consistency.
3. If there is only 1 encoding option please represent that here. I do not mind including the option for consistency and to communicate to the user the encoding type.

Reply:
These were updated to not show `default: "default"` or the encodings list if the only option is `default`.